web/src/components/filings-calendar/FilingsCalendar.test.tsx

```diff
@@ -719,13 +729,13 @@ describe("<FilingsCalendar />", () => {
     });
 
     it("doesn't display past months in list view", () => {
-      const pastDueDate = getCurrentDate().subtract(1, "months");
+      const pastDueDate = getAprilDateOfThisYear().subtract(1, "months");
```

Comment:
[sand] Ideally, I'd love if this also covered the edge case of the prior year, which is why I was interested when you and @adidner were talking about setting the month to January initially. It's not super important though, and I think this is sufficient to cover our bases for now, but something worth considering if we come back to this test file again.

Reply:
I'm not sure I understand what you mean here. Do you mean adding another test for looking at the prior year? 

This works better with April because if you did January and then subtracted a month you would still be cut off but by the fact you were in the prior calendar year rather than being in the current calendar year but already past that due date 

So the yes would pass but you wouldn't be testing what you wanted.